proposals/NNNN-explicit-padding.md

```diff
@@ -108,12 +117,32 @@ cbuffer cb0 : register(b0) {
 }>
 ```
 
+For structs, we add padding to align them as appropriate:
+
+```hlsl
+struct S {
+  int v;
+};
+cbuffer cb1 : register(b0) {
+  int i; // offset   0,  size 4  (+12)
+  S s;   // offset  16,  size 4
+  int j; // offset  20,  size 4
+}
+
+```llvm
+%__cblayout_cb1 = type <{
+  i32, target("dx.Padding", 12),
+  i32,
```

Comment:
Are we flattening the struct `S` here? I though we preserve it.

Reply:
The struct is preserved. I've fixed the example.